### PR TITLE
[CHORE] Remove Buds on Platforms

### PR DESCRIPTION
**Pull request by elias7905**
# Description

This PR is a continuation from PR #2944 
1.  Removes the Buds on the platforms

Before:
image
After:
image

2.  Added a button that links to the Buds Collection on Opensea _(This menu won't show up if you own a bud)_
3.  Read More button links to the Buds page on whitepaper
image

Fixes #issue

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

# How Has This Been Tested?

Ran the program on local host, make sure the buttons links to the webpages that I linked in the code.

# Checklist:

- [x] Title of the PR is relevant and is prefixed with [FEAT], [CHORE] or [FIX]
- [x] Screenshot if it includes any UI changes
- [x] I have read the contributing guidelines and agree to the T&Cs
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: src/features/game/expansion/components/Water.tsx [hunk 1]
```diff
@@ -9,8 +9,6 @@ import cossies from "assets/decorations/cossies.png";
 import pirateIsland from "assets/land/desert_island.webp";
 import bearIsland from "assets/land/bear_island.webp";
 import abandonedLand from "assets/land/abandoned_land.webp";
-import snowBudRaft from "assets/land/snow_raft.gif";
```
Comment (brian-man) [inline src/features/game/expansion/components/Water.tsx:12]: Remove the art as well?

Comment (elias7905) [reply src/features/game/expansion/components/Water.tsx:12]: @brian-man as in the assets files?

**Comment by elias7905** [comment]
I need some help linking the "Read More" Button to the Bud NFT docs
Anyone able to guide me?

**Review by adamhannigan — APPROVED** [review]
LGTM

**Pull request merged**